### PR TITLE
Kubecon landing page

### PR DESCRIPTION
**Pull request by polfliet**
## Description
Adding a landing page for Kubecon Europe 2020

## Reviewer Notes
I took the same approach as the developer champion page. Later we could refactor to a more generic Events page and have subpages for each event.

## Screenshot(s)
Screenshot 2020-08-13 at 10 51 39

### CODE REVIEW

File: src/components/NavigationItems.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ const iconLibrary = {
   'Automate workflows': 'automation',
   'Explore docs': 'developerDocs',
   'Developer champions': 'developerChampions',
+  'Kubecon & CloudNativeCon': 'event',
```
Comment (djsauble) [inline src/components/NavigationItems.js:18]: It feels a bit odd to have this be a top-level nav item, given that this is a temporary page (and the other top-level nav items are permanent).

Suggest having a top-level "Attend events" nav item with Kubecon nestled underneath.

File: src/pages/kubecon-europe-2020.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+          <section className={cx(styles.section, styles.twoColumn)}>
+            <div>
+              <h3>
+                New Relic welcomes you at <br />
+                Virtual Kubecon and CloudNativeCon Europe 2020!
+              </h3>
+              <p>
+                Learn more about the New Relic One platform, the only
+                observability platform that provides open, connected and
+                programmable observability for cloud-native environments. Join
```
Comment (djsauble) [inline src/pages/kubecon-europe-2020.js:27]:
```suggestion
                programmable observability for cloud-native environments.
```

File: src/pages/kubecon-europe-2020.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+            <img
+              className={styles.img}
+              src={kubeconHeader}
+              alt="KubeCon and CloudNativeCon Europe 2020"
+            />
+          </section>
+          <section className={styles.section}>
+            <h2>Attend one of our lightning talks</h2>
+          </section>
+          <section className={cx(styles.section, styles.assetTable)}>
```
Comment (djsauble) [inline src/pages/kubecon-europe-2020.js:52]: When sequence matters, as it does here, grids are a poor choice. I suggest using a table or list instead. For example:

| What | Who | When |
| ----- | ----- | ------ |
| We handle Prometheus, you keep Grafana | Samuel Vandamme | Mon Aug 17 @ 14:35 CEST |
| How to use and customize Helm charts | Douglas Camata | Mon Aug 17 @ 16:25 CEST |
| ... | ... | ... |

File: src/pages/kubecon-europe-2020.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            <div className={styles.highlightDiv}>
+              <FeatherIcon
+                className={styles.pointIcon}
+                name="search"
+                size="2rem"
+              />
+              <h4 className={styles.highlightText}>
+                Go to the virtual expo tab, and find New Relic in Silver Hall B
+                to attend a lightning talk.
+              </h4>
```
Comment (djsauble) [inline src/pages/kubecon-europe-2020.js:103]: This is important info! I'd suggest putting it at the top of this section and using the callout component. See an example here: https://developer.newrelic.com/build-apps/add-time-picker-guide

Screen Shot 2020-08-13 at 9 23 48 AM

File: src/pages/kubecon-europe-2020.js [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+            </div>
+          </section>
+          <section className={styles.section}>
+            <h2>Want some action now? Check out the following videos!</h2>
```
Comment (djsauble) [inline src/pages/kubecon-europe-2020.js:107]: Given this title, I'd expect these to be previews of the lightning talks that people will see at the conference.

Is that true? If not, I'd suggest making it clear that these videos are unrelated to the conference. For example:

`Start your learning journey before Kubecon starts`

File: src/components/NavigationItems.js (hunk 1 above)
Comment (jpvajda) [reply src/components/NavigationItems.js:18]: Yes Sir! agreed we need to add that the side nav

File: src/pages/kubecon-europe-2020.js (hunk 5 above)
Comment (polfliet) [reply src/pages/kubecon-europe-2020.js:107]: Yes these videos are related

File: src/pages/kubecon-europe-2020.js (hunk 4 above)
Comment (polfliet) [reply src/pages/kubecon-europe-2020.js]: Done, added a note at the top of the section

File: src/pages/kubecon-europe-2020.js (hunk 3 above)
Comment (djsauble) [reply src/pages/kubecon-europe-2020.js:57]: We don't need to be fancy, a simple Markdown table for now should suffice until we can componentize it properly.

**Comment by djsauble** [comment]
Once we swap the events over to a Markdown table I'm good!

File: src/pages/kubecon-europe-2020.js (hunk 3 above)
Comment (polfliet) [reply src/pages/kubecon-europe-2020.js:57]: The page is created in react, I don't know how to include markdown there.
I tried with an html table but that looks horrible. Actually using a markdown table on an .mdx page also doesn't look good in my opinion.
I am sorry but this goes beyond my skills and I just want get this out so I am hoping someone can help?
